modules/virt-using-supported-boot-source-names.adoc

```diff
@@ -0,0 +1,46 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virtual_disks/virt-creating-and-using-default-os-images.adoc
+
+[id="virt-using-supported-boot-source-names_{context}"]
+= Using supported boot source names
+
+When you upload boot sources and associate them with an operating system image, it is important that the volume name match the name of the system-defined template. If you do not name the volume in this manner, you will be unable to upload the image and use it to create virtual machines.
+
+// What should the user do if their OS is not listed in the following table? // Should we say only the following OSs are supported?
+
+Use the following table to determine the name of the volume that you define as a boot source.
+
+[cols="2",options="header"]
+|===
+|To create a boot source for this OS:
+|Name the volume:
+
+|Fedora
+|fedora32
+
+|Centos 7
+|centos7
+
+|Centos 8
+|centos8
+
+|Red Hat Enterprise Linux 8
+|rhel8
+
+|Red Hat Enterprise Linux 7
+|rhel7
+
+|Open SUSE
+|opensuse15
+
+|Ubuntu
+|ubuntu18
+
+|Windows 10
+|win10
+
+|Windows 2019 Server
+|win2k19
+
+|===
```

Comment:
Don't see dv name for RHEL6, Windows 2016 and Windows 2012 R2

Reply:
Added.